src/router/routes.js

```diff
@@ -290,6 +290,14 @@ module.exports = [
 			excludeFromStaticSitemap: true,
 		}
 	},
+	{
+		path: '/lending-home',
```

Comment:
Looks like we need a PR to `puppet` for this new route.

Reply:
It will be good to create this route in puppet too. But if we go the client only approach to the redirect it would still work b/c of router push navigating in page instead of refreshing...

Reply:
@mcstover Would you rather do the client-side redirect instead of the server-side that I mentioned in another comment?

Reply:
We will need to update .htaccess in the kiva/kiva repo to handle this route in the dev-vm and create a new ui route in puppet. I can help with the puppet portion as needed.